libr/debug/p/native/linux/linux_debug.c

```diff
@@ -97,6 +97,68 @@ int linux_handle_signals(RDebug *dbg, int tid) {
 		// siginfo.si_code -> HWBKPT, USER, KERNEL or WHAT
 		// TODO: DO MORE RDEBUGREASON HERE
 		switch (dbg->reason.signum) {
+		case SIGABRT: // 6 / SIGIOT // SIGABRT
+			dbg->reason.type = R_DEBUG_REASON_SIGNAL;
+			break;
+		case SIGALRM:
+			dbg->reason.type = R_DEBUG_REASON_SIGNAL;
+			break;
+		case SIGBUS:
+			dbg->reason.type = R_DEBUG_REASON_SIGNAL;
+			break;
+		case SIGCHLD:
+			dbg->reason.type = R_DEBUG_REASON_SIGNAL;;
+			break;
+		case SIGCONT:
+			dbg->reason.type = R_DEBUG_REASON_SIGNAL;
+			break;
+#if __sparc__
+		case SIGEMT:
+			dbg->reason.type = R_DEBUG_REASON_SIGNAL;
+			break;
+#endif
+		case SIGFPE:
+			dbg->reason.type = R_DEBUG_REASON_SIGNAL;
+			break;
+		case SIGHUP:
+			dbg->reason.type = R_DEBUG_REASON_SIGNAL;
+			break;
+		case SIGILL:
+			dbg->reason.type = R_DEBUG_REASON_SIGNAL;
+			break;
+		case SIGPWR:
+			break;
+		case SIGINT:
+			dbg->reason.type = R_DEBUG_REASON_USERSUSP;
+			break;
+		case SIGIO:
```

Comment:
why empty case. this is introducing more bugs. please just remove all the cases and onlyhandle the ones that are not just a signal